spec.bs

```diff
@@ -727,12 +727,18 @@ A <dfn export for=fencedframetype>pending event</dfn> is a [=struct=] with the f
 
   : <dfn>event</dfn>
   :: a [=fencedframetype/destination event=]
+
+  : <dfn>request initiator</dfn>
+  :: an [=origin=]
 </dl>
 
 A <dfn export for=fencedframetype>reporting destination info</dfn> is a [=struct=] with the
 following [=struct/items=]:
 
 <dl export dfn-for="reporting destination info">
+  : <dfn>reporting url declarer origin</dfn>
+  :: null, or an [=origin=]
```

Comment:
When is this null? Maybe I missed it but doesn't seem to be set to null anywhere.

Reply:
On [the implementation side](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/fenced_frame/fenced_frame_reporter.h;l=371-378;drc=27da6a274be3aaa246d3397d3bd8595f581e2820) of things, the optional value is serving the same purpose that the [pending events or reporting destination info in the fenced frame reporting map](https://wicg.github.io/fenced-frame/#fencedframetype-fenced-frame-reporting-map) is doing. Namely, marking whether the metadata has been finalized yet. There's no need to have it be null in the spec, so I'll make it just an origin.